torch/_functorch/config.py

```diff
@@ -4,26 +4,31 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Callable
-
 
 """
 Global flags for aot autograd
 """
 
 import os
 import sys
-from typing import Literal, Optional, TYPE_CHECKING
+from abc import ABC, abstractmethod
+from typing import Any, Callable, Literal, Optional, TYPE_CHECKING, Union
+from typing_extensions import TypeAlias
 
+import torch
 from torch.utils._config_module import Config, install_config_module
 
 
 # [@compile_ignored: debug]
-_save_config_ignore = [
+_save_config_ignore: list[str] = [
     # callable not serializeable
     "joint_custom_pass",
 ]
 
+_cache_config_ignore_prefix: list[str] = [
```

Comment:
I see inductor custom passes are all included in the `_cache_config_ignore_prefix` 

following the practice here.